documentation/library_guide/parallel_api/execution_policies.rst

```diff
@@ -1,65 +1,70 @@
-                                  all functions provided are SIMD-safe.
---------------------------------- ------------------------------
-``par``                           Parallel execution by multiple threads.
---------------------------------- ------------------------------
-``par_unseq``                     Combined effect of ``unseq`` and ``par``.
---------------------------------- ------------------------------
-``dpcpp_default``                 Massive parallel execution on devices using |dpcpp_short|.
---------------------------------- ------------------------------
-``dpcpp_fpga``                    Massive parallel execution on FPGA devices.
-================================= ==============================
+According to `the oneAPI specification
+<https://uxlfoundation.github.io/oneAPI-spec/spec/elements/oneDPL/source/index.html>`_,
+|onedpl_long| (|onedpl_short|) provides execution policies semantically aligned with the C++ standard,
+also referred to as *standard aligned* or *host execution policies*, and the *device execution policies*
+to run data parallel computations on heterogeneous systems.
+
+The execution policies are defined in the ``oneapi::dpl::execution`` namespace and provided
+in the ``oneapi/dpl/execution`` header. The policies have the following meaning:
+
+====================== =====================================================
+Policy Value or Type   Description
+====================== =====================================================
+``seq``                The standard aligned policy for sequential execution.
+---------------------- -----------------------------------------------------
+``unseq``              The standard aligned policy for unsequenced SIMD execution.
+                       This policy requires user-provided functions to be SIMD-safe.
+---------------------- -----------------------------------------------------
+``par``                The standard aligned policy for parallel execution by multiple threads.
+---------------------- -----------------------------------------------------
+``par_unseq``          The standard aligned policy with the combined effect of ``unseq`` and ``par``.
+---------------------- -----------------------------------------------------
+``device_policy``      The class template to create device policies for data parallel execution.
+---------------------- -----------------------------------------------------
+``dpcpp_default``      The device policy for data parallel execution on the default SYCL device.
+---------------------- -----------------------------------------------------
+``fpga_policy``        The class template to create policies for execution on FPGA devices.
+---------------------- -----------------------------------------------------
+``dpcpp_fpga``         The device policy for data parallel execution on a SYCL FPGA device.
+====================== =====================================================
```

Comment:
I agree that it's bad having the fpga policies for so many years and not list them in the documentation, on the other hand, we were discussing that we probably want to revise the names of our global policies. Having them documented would require us to go through deprecation way, because it will be a breaking change. How do you feel about leaving the status quo (before changes in the PR)?

Reply:
Got it, thanks